src/main/resources/com/google/api/codegen/php/test.snip

```diff
@@ -67,128 +67,179 @@
 @end
 
 @private nonStreamingOptionalArrayTestCase(test)
+    @switch test.clientMethodType
+    @case "OptionalArrayMethod"
+        {@simpleTestCase(test)}
+        {@simpleExceptionTestCase(test)}
+    @case "PagedOptionalArrayMethod"
+        {@pageStreamingTestCase(test)}
+        {@simpleExceptionTestCase(test)}
+    @case "OperationOptionalArrayMethod"
+        {@lroTestCase(test)}
+        {@simpleExceptionTestCase(test)}
```

Comment:
Why is the LRO exception test the same as the others? I assume the test should be different because we should test the error handling of the operation object.